### PR TITLE
Add Copyright for ticket: ZQS-1162

### PR DESCRIPTION
**Pull request by mstechly**
## Description

This PR updates the missing copyright notices.

### CODE REVIEW

File: src/orquestra/quantum/operators/_openfermion_utils/config.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
```
Comment (max-radin) [inline src/orquestra/quantum/operators/_openfermion_utils/config.py:2]: This file already has a copyright notice

**Review by max-radin — CHANGES_REQUESTED** [review]
There's a few files that already had copyright notices, see comments.

File: src/orquestra/quantum/operators/_openfermion_utils/config.py (hunk 1 above)
Comment (mstechly) [reply src/orquestra/quantum/operators/_openfermion_utils/config.py]: Fixed in https://github.com/zapatacomputing/orquestra-quantum/pull/53/commits/98a53a9b5102d794e3b8189caaa4e6db7f364a38

File: src/orquestra/quantum/operators/_openfermion_utils/operator_utils.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
```
[the remark this answers is not in the document]
Comment (mstechly) [reply src/orquestra/quantum/operators/_openfermion_utils/operator_utils.py]: Fixed in https://github.com/zapatacomputing/orquestra-quantum/pull/53/commits/98a53a9b5102d794e3b8189caaa4e6db7f364a38

File: src/orquestra/quantum/operators/_openfermion_utils/sparse_tools.py [hunk 3]
```diff
@@ -1,3 +1,6 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
```
[the remark this answers is not in the document]
Comment (mstechly) [reply src/orquestra/quantum/operators/_openfermion_utils/sparse_tools.py]: Fixed in https://github.com/zapatacomputing/orquestra-quantum/pull/53/commits/98a53a9b5102d794e3b8189caaa4e6db7f364a38

File: src/orquestra/quantum/operators/_pauli_operators.py [hunk 4]
```diff
@@ -1,3 +1,6 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
```
[the remark this answers is not in the document]
Comment (mstechly) [reply src/orquestra/quantum/operators/_pauli_operators.py]: Fixed in https://github.com/zapatacomputing/orquestra-quantum/pull/53/commits/98a53a9b5102d794e3b8189caaa4e6db7f364a38

**Pull request merged**